lib-utilities/config/odimra_config.json

```diff
@@ -128,5 +128,10 @@
 		  "ConnectionMethodType": "Redfish",
 		  "ConnectionMethodVariant":"Compute:BasicAuth:URP_v1.0.0"
 	  }
-   ]
+  ],
+  "EventConf": {
+		"DeliveryRetryAttempts" : 3,
+		"DeliveryRetryIntervalSeconds" : 60,
+		"RetentionEventsInMinutes" : 10
```

Comment:
Rename it to `RetentionOfUndeliveredEventsInMinutes`

Reply:
changed